Documentation/Groups/Groups.md

```diff
@@ -0,0 +1,105 @@
+# Using Groups
+
+A group item in a workbook allows you to logically group a set of steps in a workbook. In reading mode, the group itself has no "chrome", only showing the items that are inside the group:
+
+![group in reading mode](../Images/groups-view.png)
+
+When editing the workbook, you can see those 2 items are acutally inside a group item:
+
+![group in reading mode](../Images/groups-edit.png)
+
+In the screenshot above, the group ("group in a workbook") is in edit mode being edited, showing it contains 2 items (inside the dashed area), and the text step inside the group ("text in a group") is also being edited. a query step appears below the text step in the group 
+
+
+Groups in workbooks are useful for several things:
+
+1. layout - in scenarios where you want items to "stack up" vertically, you can create a group of items that will all stack up, and set the styling of the *group* to be a percentage width, instead of setting percentage width on all the individual items.
+
+2. visibility - in scenarios where you want many items to hide/show together, you can set the visibility of the entire group of items, isntead of setting visibility settings on each individual item. This can be very useful in templates that use tabs, as you can use a group as the content of the tab, and the entire group can be hidden/shown based on a parameter set by the selected tab.
+
+3. performance - in cases where you have a very large template, with many tabs, etc, you can use groups to split up a single template into many templates, and each tab of the workbook can be a group that loads all of its content from that other template. In these cases, the contents of the group won't load or run until a user makes that group visible
```

Comment:
The wording sounds a bit hard to understand here, maybe something like this sounds easier to understand?

`in cases where you have a very large templates with many sections or tabs, you can convert each section into it's own sub-template and use groups to load all the sub-templates within the top level template. In these cases, the contents of the sub-templates won't load or run until a user makes those groups visible. See [How to split a large template into many templates](#how-to-split-a-large-template-into-many-templates) for more information.`